rust-sdk/whirlpool/src/create_pool.rs

```diff
@@ -41,6 +41,60 @@ pub struct CreatePoolInstructions {
     pub additional_signers: Vec<Keypair>,
 }
 
+/// Creates the necessary instructions to initialize a Splash Pool.
+///
+/// # Arguments
+///
+/// * `rpc` - A reference to a Solana RPC client for communicating with the blockchain.
+/// * `token_a` - The public key of the first token mint address to include in the pool.
+/// * `token_b` - The public key of the second token mint address to include in the pool.
+/// * `initial_price` - An optional initial price of token A in terms of token B. Defaults to 1.0 if not provided.
+/// * `funder` - An optional public key of the account funding the initialization process. Defaults to the global funder if not provided.
+///
+/// # Returns
+///
+/// A `Result` containing `CreatePoolInstructions` on success:
+/// * `instructions` - A vector of Solana instructions needed to initialize the pool.
+/// * `initialization_cost` - The estimated rent exemption cost for initializing the pool, in lamports.
+/// * `pool_address` - The public key of the newly created pool.
+/// * `additional_signers` - A vector of `Keypair` objects representing additional signers required for the instructions.
+///
+/// # Errors
+///
+/// This function will return an error if:
+/// - The funder account is invalid.
+/// - Token mints are not found or have invalid data.
+/// - The token mint order does not match the canonical byte order.
+/// - Any RPC request to the blockchain fails.
+///
+/// # Example
+///
+/// ```rust
+/// use solana_client::rpc_client::RpcClient;
+/// use solana_sdk::{pubkey::Pubkey, signer::Keypair};
+/// use orca_whirlpools::create_splash_pool_instructions
+/// use std::str::FromStr;
+///
+/// set_whirlpools_config_address(WhirlpoolsConfigInput::SolanaDevnet).unwrap()
```

Comment:
`.unwrap` in rust results in a `panic` which works different from throwing in ts. A panic stops the code and cannot be recovered.

I guess for script-style code this is fine (since you just want your script to stop on an error) but for production-environments you generally want to handle errors.